### DISCUSSION TITLE
Pimcore Academy - Working with Object Advanced - What are possible ways for attaching additional information to relations?

### DISCUSSION THREAD

**Discussion by michaelopre** [pimcore/pimcore#15308] [Q&A / Support] [answered]
Hi, 

its me again. Sorry, for claiming the community so much, but I have no other idea to discuss this topics to understand.

The Question What are possible ways for attaching additional information to relations?
1. Use a field collection with many-to-one relation and additional data attributes.
2. Use simple many-to-many relation type.
3. Use advanced many-to-many relation type.
4. This requirement cannot be modelled with Pimcore.
5. Use block with many-to-one relation and additional data attributes.

Is a little bit confusing in my eyes. If I want to attach additional information to a relation, the only ways are to add field collections or blocks to the relation to use additional data attributes. Using a relation like the simple many-to-many relation is just a relation. That means a relation without any additional information.

Or do I misunderstand the complete question, because the relation is already the additional information and there is no way to add any additional information!?

Thank you for your help.

Best,
Michael

**Comment by dvesh3** [comment]
What do you think about **advanced many-to-many relation type** as this allows to add meta data to a relation so do you think this can this be an option too?

**Reply by michaelopre** [reply, marked as answer]
Thank you! If I understand you right, it is:

- Use a field collection with many-to-one relation and additional data attributes.
- Use advanced many-to-many relation type. -> because of the meta data
- Use block with many-to-one relation and additional data attributes.
